files/logstash-patterns/dpkg

```diff
@@ -5,7 +5,7 @@ DPKG_PKGARCH %{DPKG_PACKAGE:package}(:%{DPKG_ARCH:arch})?
 DPKG_VERSION [-+~<>\.0-9a-zA-Z]+
 DPKG_ACTION0 (startup archives|startup packages|conffile)
 DPKG_ACTION1 (configure|remove|upgrade|purge|status (config-files|installed|not-installed|triggers-awaited|triggers-pending|unpacked|half-installed|half-configured))
-DPKG_ACTION2 (trigproc|upgrade)
+DPKG_ACTION2 (install|trigproc|upgrade)
```

Comment:
What example log line will this change add support for? The `installed` match in `DPKG_ACTION1` seems sufficient to me.